README.md

```diff
@@ -1,214 +1,201 @@
 
-The new home for all @openfn
-[language adaptors](https://docs.openfn.org/adaptors) - open-source Javascript
-or Typescript modules that provide helper functions to communicate with a
-specific external system.
-
-For a fully open source workflow automation platform that leverages these
-adaptors, see [OpenFn Lightning](https://github.com/OpenFn/lightning).
-
-## Getting Started
-
-_Note: [asdf](https://github.com/asdf-vm/asdf) to be installed globally on your
-machine. Add the nodejs and pnpm plugin once asdf is installed globally._
-
-A few first time repo steps:
-
-```
+Open-source JavaScript/TypeScript modules that provide helper functions to
+communicate with external systems. These adaptors are used by
+[OpenFn Lightning](https://github.com/OpenFn/lightning) and the
+[OpenFn CLI](https://github.com/openfn/cli) for workflow automation.
+
+# Table of Contents
+
+- [Quick Start](#quick-start)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [Using Adaptors](#using-adaptors)
+  - [With OpenFn CLI](#with-openfn-cli)
+  - [With OpenFn Lightning](#with-openfn-lightning)
+- [Contributing](#contributing)
+- [Development](#development)
+  - [Creating a New Adaptor](#creating-a-new-adaptor)
+    - [Prerequisites](#prerequisites-1)
+    - [Development Steps](#development-steps)
+    - [Testing Documentation Changes](#testing-documentation-changes)
+    - [Best Practices](#best-practices)
+- [Changesets](#changesets)
+- [Versioning](#versioning)
+- [Releases](#releases)
+- [Pre-releases](#pre-releases)
+- [Metadata](#metadata)
+- [Useful Resources](#useful-resources)
+
+## Quick Start
+
+### Prerequisites
+
+- [asdf](https://github.com/asdf-vm/asdf) installed globally
+- `Node.js` and `pnpm` managed via asdf
+
+### Setup
+
+```bash
+# Install asdf plugins
 asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
 asdf plugin-add pnpm
-```
 
-Then:
-
-```
-asdf install # Install tool versions
+# Install dependencies and build
+asdf install
 pnpm install
 pnpm build
 pnpm run setup
 ```
 
-## Running scripts
+## Using Adaptors
 
-Every repo provides a common set of npm scripts:
+### With OpenFn CLI
 
-To run them for all scripts in `packages`, call
-`pnpm --filter "./packages/** <script>`.
+1. Install the CLI globally:
 
-For example:
+   ```bash
+   npm install -g @openfn/cli
+   ```
 
-```
-pnpm --filter "./packages/**" build
-pnpm --filter "./packages/**" test
-```
+2. Create a job file (e.g., `job.js`):
 
-## Contributing
+   ```javascript
+   // Example: Send data to Salesforce
+   create('Account', {
+     Name: $.data.company_name,
+     Industry: $.data.industry,
+   });
+   ```
 
-### Assign yourself to an issue
+3. Run the job:
+   ```bash
+   openfn job.js -a salesforce -s tmp/state.json -o tmp/output.json
```

Comment:
But this won't use the monorepo. We don't need to document the CLI here, we need to document local adaptor usage (which happens to use the CLI)